Source/Core/DolphinQt2/HotkeyScheduler.cpp

```diff
@@ -213,29 +213,32 @@ void HotkeyScheduler::Run()
               IsHotkey(HK_TRIGGER_SYNC_BUTTON, true));
       }
 
-      if (IsHotkey(HK_STEP))
-        emit Step();
+      if (SConfig::GetInstance().bEnableDebugging)
```

Comment:
Maybe move this out into another function so you can just replace this long indented region with a call to ``CheckNextHotkey()``.